src/cfnlint/data/ExtendedSpecs/all/01_spec_patch.json

```
@@ -108,5 +109,20 @@
     "op": "replace",
     "path": "/PropertyTypes/AWS::Backup::BackupPlan.LifecycleResourceType/Properties/MoveToColdStorageAfterDays/PrimitiveType",
     "value": "Long"
+  },
+  {
+    "op": "replace",
+    "path": "/ResourceTypes/AWS::WAFv2::RegexPatternSet/Properties/RegularExpressionList",
+    "value": {
+      "Documentation": "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-wafv2-regexpatternset-regularexpressionlist.html#cfn-wafv2-regexpatternset-regularexpressionlist-regularexpressionlist",
+      "ItemType": "Regex",
```

```suggestion
      "PrimitiveItemType": "String",
```

still getting: `E3002: Expecting an object at Resources/RegexPatternSet/Properties/RegularExpressionList/0`

with this deployable template:

https://github.com/PatMyron/cloud/blob/3a08e527688da6a5b06ff9c2eaa873db31481783/cloudformation/minimal.yaml#L658-L664

```yaml
Resources:
  RegexPatternSet:
    Type: AWS::WAFv2::RegexPatternSet
    Properties:
      Name: Name
      Scope: REGIONAL
      RegularExpressionList:
        - RegularExpression
```

---

Although I'm not sure if the `RegexString` property is also valid or not